lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -4876,6 +4876,16 @@ instance IsServerError ErrPostTx where
                 , "node. Here's an error message that may help with "
                 , "debugging:\n", err
                 ]
+        ErrPostTxMempoolFull ->
+            apiError err425
+            {errBody = "Mempool is full, please try resubmitting again later."}
+                MempoolIsFull $ mconcat
+                [ "The submitted transaction was rejected by the external "
+                , "blockchain provider because mempool was full."
```

Comment:
```suggestion
                [ "The submitted transaction was rejected by the Cardano "
                , "node because its mempool was full."
```

Reply:
Here again, see my  comment on the exclusivity of the error below